.circleci/config.yml

```diff
@@ -146,7 +144,7 @@ jobs:
           /app/scripts/entrypoint.pl load_test_data --legacy
       - run:
           command: |
-            /app/scripts/entrypoint.pl test_selenium | tee artifacts/$CIRCLE_JOB.txt
+            false "#/app/scripts/entrypoint.pl test_selenium | tee artifacts/$CIRCLE_JOB.txt"
```

Comment:
this doesn't look correct

Reply:
Needed to stub it out to verify how circleci works; the local circleci client doesn't entirely behave the same way. :)